cmake/SearchForStuff.cmake

```diff
@@ -145,7 +145,7 @@ else()
 	check_lib(SAMPLERATE samplerate samplerate.h)
 
 	if(NOT QT_BUILD)
-		check_lib(SDL2 SDL2 SDL.h PATH_SUFFIXES SDL2)
+		find_optional_system_library(SDL2 3rdparty/sdl2 2.0.12)
```

Comment:
Wrong version here.. Should be 2.0.22, not 12.

Reply:
2.0.12 is the minimum version needed for wx builds
Though we could require 2.0.22 anyways for better controller support

Reply:
Ah, alright, didn't realise we weren't going to 22 across the board (though with wx going away, we will soon enough anyways).

Reply:
Can verify that SDL2 does not build with -DUSE_SYSTEM_SDL2=ON on a qt build with sdl lower than 2.0.22 then, in which case I've successfully checked all the suggested testing steps. I just initially tried it with a wx build and sdl 2.0.20 and it compiled...